spec/requests/sign_up_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'swagger_helper'
+
+RSpec.describe 'sign_up', type: :request do
+  path '/sign_up' do
+    post('create sign_up') do
+      tags 'Sign_up'
+      produces 'application/json'
+      parameter name: :signup, in: :body, schema: {
+        type: :object,
+        properties: {
+          name: { type: :string },
+          phone: { type: :string },
+          gender: { type: :string, enum: %w[male female other] },
+          email: { type: :string },
+          birthday: { type: :string },
```

Comment:
Could you leave an exampe for this request as it is not clear what format for the date is used?

Reply:
Added date format example